src/Client/Updater.php

```diff
@@ -163,14 +165,14 @@ public function refresh() : bool
                 "$snapShotVersion.snapshot.json",
                 static::MAXIMUM_DOWNLOAD_BYTES
             );
+            // TUF-SPEC-v1.0.9 Section 5.3.1
```

Comment:
Given the fact that the numbering does change, I wonder if we should:
- Simply link the relevant spec item
- Say in words what the step is (the bolded part in the spec doc)

Reply:
Let's ignore this in this PR and fix later once we have a response to https://github.com/theupdateframework/specification/pull/133.